### ISSUE TITLE
Update components table to include deployed components for each patch release

### ISSUE THREAD

**Issue by a-mccarthy** [goharbor/website#235]
We should make sure to document which components are deployed or required for each patch version of harbor.

We currently document this per minor version, but not for the patch version, https://goharbor.io/docs/2.3.0/install-config/#harbor-components

**Comment by a-mccarthy** [comment]
Do we also need to include patch versions in the table for building from source, https://goharbor.io/docs/2.3.0/build-customize-contribute/compile-guide/?

**Comment by a-mccarthy** [comment]
cc: @yanji09

**Comment by yanji09** [comment]
> Do we also need to include patch versions in the table for building from source, https://goharbor.io/docs/2.3.0/build-customize-contribute/compile-guide/?

Good point. But the component versions recommended here are for building harbor code. Generally, versions will not be changed in patch release. For now, just keep it as is.

**Comment by yanji09** [comment]
Orlin, would you help on this during Abbie's absence?

**Comment by OrlinVasilev** [comment]
@yanji09 - are these true for release 2.3? 

Postgresql | 9.6.10-1.ph2
Redis | 4.0.10-1.ph2
Beego | 1.9.0
Chartmuseum | 0.9.0
Docker/distribution | 2.7.1
Docker/notary | 0.6.1
Helm | 2.9.1
Swagger-ui | 3.22.1

Or they need update?

**Comment by yanji09** [comment]
They need update. You can check with Yan about the details.

**Comment by OrlinVasilev** [comment]
Hi @wy65701436, can you please provide update on the versions :) thank you!

**Comment by OrlinVasilev** [comment]
@wy65701436 - are you active on this one ?

**Comment by a-mccarthy** [comment]
@wy65701436 are you willing to help review the component versions from Orlin's comment [above](https://github.com/goharbor/website/issues/235#issuecomment-885610583)? These were for 2.3, have they changed since? Thanks!

>Postgresql | 9.6.10-1.ph2
Redis | 4.0.10-1.ph2
Beego | 1.9.0
Chartmuseum | 0.9.0
Docker/distribution | 2.7.1
Docker/notary | 0.6.1
Helm | 2.9.1
Swagger-ui | 3.22.1
